Issue #21272 fix and updating workflow to include access assignment. (#21631)
microsoftgraph/microsoft-graph-docs: api-reference/beta/resources/delegatedadminrelationships-api-overview.md

`https://admin.microsoft.com/AdminPortal/Home#/partners/invitation/granularAdminRelationships/{adminRelationshipID}`

> Send the invitation link to the customer for them to approve the GDAP request. For example, `https://admin.microsoft.com/AdminPortal/Home#/partners/invitation/granularAdminRelationships/5d027261-d21f-4aa9-b7db-7fa1f56fb163-8777b240-c6f0-4469-9e98-a3205431b836` is an invitation link, where `5d027261-d21f-4aa9-b7db-7fa1f56fb163-8777b240-c6f0-4469-9e98-a3205431b836` is the admin relationship request ID. The GDAP relationship gets into **active** state post the customer approval.

Reviewer: ```suggestion
Send the invitation link to the customer for them to approve the GDAP request. For example, `https://admin.microsoft.com/AdminPortal/Home#/partners/invitation/granularAdminRelationships/5d027261-d21f-4aa9-b7db-7fa1f56fb163-8777b240-c6f0-4469-9e98-a3205431b836` is an invitation link, where `5d027261-d21f-4aa9-b7db-7fa1f56fb163-8777b240-c6f0-4469-9e98-a3205431b836` is the admin relationship request ID. Once the customer approves the GDAP request, the GDAP relationship will transition to the active state.
```

> Next, to complete the workflow for enabling AOBO management of customer tenant, create new access assignment for this delegated admin relationship using [Create accessAssignments](../api/delegatedadminrelationship-post-accessassignments.md) API. 

Reviewer: ```suggestion
To finalize the workflow for enabling AOBO (Admin-on-behalf-of) management of the customer's tenant, proceed by creating a new access assignment for the delegated admin relationship using the [Create accessAssignments](../api/delegatedadminrelationship-post-accessassignments.md) API. 
```

> ### Lifecycle of a GDAP Relationship Access Assignment

Reviewer: ```suggestion
### Lifecycle of a GDAP relationship access assignment
```

> The status of the Delegated Admin access assignment. The status transitions are as follows:

Reviewer: ```suggestion
The status of the delegated admin access assignment. The status transitions are as follows:
```

> ![Delegated Admin access assignment status transition diagram](access-assignment-status-transitions.png)

Reviewer: ```suggestion
![Delegated admin access assignment status transition diagram](access-assignment-status-transitions.png)
```

> ![Delegated admin access assignment status transition diagram](access-assignment-status-transitions.png)

Reviewer 2: Global: The font color and size in the diagrams in these topics is faint and difficult to read. Consider updating the diagrams to be more readable. You can preview the topic here: https://review.learn.microsoft.com/en-us/graph/api/resources/delegatedadminrelationships-api-overview?view=graph-rest-beta&branch=pr-en-us-21631

Also note that for accessibility, any information conveyed in a diagram also needs to be available to a screen reader. In addition to providing the diagrams, please add text that describes the transitions. Note that readers who rely on a screen reader won't be able to read the numbers from the diagram and what the numbered lists correlate to won't be clear, so it will be helpful to elaborate.
